.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -460,6 +463,16 @@ def write(x=''):
             }
         }
 
+        if args.setup_for_sample_courses:
+            worker_dict['primary']['capabilities'].extend([
+                'cpp',
+                'python',
+                'matlab',
```

Comment:
This should be removed from the set of capabilities here. By default, the sample courses does not include matlab example as it requires [extra steps](https://submitty.org/developer/configuring_tie_in_programs#matlab) to enable. We should maybe just add to that doc page adding this tag after enabling matlab.